functions/venue.js

```diff
@@ -112,9 +113,30 @@ const createVenueData = (data, context) => {
     default:
       break;
   }
+
   return venueData;
 };
 
+const createVenueData_v2 = (data, context) => ({
```

Comment:
Is this backwards compatible? If I create a venue with v2, can I edit it on admin v1?

Reply:
Not entirely.
`createVenueData` has more fields that it sends on creation, that `createVenueData_2` does.

For example, `createVenueData` sends `code_of_conduct_questions`

That data is currently not accessible nor editable.

Any data that is shared between the two functions, such as `name`, `showGrid`, `coverImageUrl` can be edited in both versions of the Admin.